azure-devops/projects/io-pay-projects/io-functions-pay-portal.tf

```
@@ -0,0 +1,161 @@
+    forks {
+      enabled       = false
+      share_secrets = false
+    }
+    override {
+      auto_cancel = false
+      branch_filter {
+        include = [var.io-functions-pay-portal.repository.branch_name]
+      }
+      path_filter {
+        exclude = []
+        include = []
+      }
+    }
+  }
+
+  repository {
+    repo_type             = "GitHub"
+    repo_id               = "${var.io-functions-pay-portal.repository.organization}/${var.io-functions-pay-portal.repository.name}"
+    branch_name           = var.io-functions-pay-portal.repository.branch_name
+    yml_path              = "${var.io-functions-pay-portal.repository.pipelines_path}/io-functions-pay-portal/code-review-pipelines.yml"
+    service_connection_id = azuredevops_serviceendpoint_github.io-azure-devops-github-pr.id
+  }
+
+  variable {
+    name         = "DANGER_GITHUB_API_TOKEN"
+    secret_value = data.azurerm_key_vault_secret.key_vault_secret["DANGER-GITHUB-API-TOKEN"].value
+    is_secret    = true
+  }
+}
+
+# code review serviceendpoint authorization
+resource "azuredevops_resource_authorization" "io-functions-pay-portal-code-review-github-ro-auth" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-ro, azuredevops_build_definition.io-functions-pay-portal-code-review, azuredevops_project.project]
+
+  project_id    = azuredevops_project.project.id
+  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-ro.id
+  definition_id = azuredevops_build_definition.io-functions-pay-portal-code-review.id
+  authorized    = true
+  type          = "endpoint"
+}
+
+resource "azuredevops_resource_authorization" "io-functions-pay-portal-code-review-github-pr-auth" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-pr, azuredevops_build_definition.io-functions-pay-portal-code-review, azuredevops_project.project]
+
+  project_id    = azuredevops_project.project.id
+  resource_id   = azuredevops_serviceendpoint_github.io-azure-devops-github-pr.id
+  definition_id = azuredevops_build_definition.io-functions-pay-portal-code-review.id
+  authorized    = true
+  type          = "endpoint"
+}
+
+# deploy
+resource "azuredevops_build_definition" "io-functions-pay-portal-deploy" {
+  depends_on = [azuredevops_serviceendpoint_github.io-azure-devops-github-rw, azuredevops_serviceendpoint_azurerm.PROD-IO, azuredevops_project.project]
+
+  project_id = azuredevops_project.project.id
+  name       = "${var.io-functions-pay-portal.repository.name}.deploy"
+  path       = "\\${var.io-functions-pay-portal.repository.name}"
+
+  repository {
+    repo_type             = "GitHub"
+    repo_id               = "${var.io-functions-pay-portal.repository.organization}/${var.io-functions-pay-portal.repository.name}"
+    branch_name           = var.io-functions-pay-portal.repository.branch_name
+    yml_path              = "${var.io-functions-pay-portal.repository.pipelines_path}/io-functions-pay-portal/deploy-pipelines.yml"
```

```suggestion
    yml_path              = "${var.io-functions-pay-portal.repository.pipelines_path}/io-functions-pay-portal/.devops/deploy-pipelines.yml"
```
https://github.com/pagopa/io-pay-portal/tree/main/io-functions-pay-portal/.devops
you need to rename deploy file